src/raids/index.ts

```diff
@@ -1,3 +1,13 @@
+type Raid = {
+  bosses: Array<Boss>,
```

Comment:
Please prefer interface over type, e.g. `interface Raid {`. Same result, different way of writing